lib/bsdlib/bsd_os.c

```diff
@@ -334,6 +334,16 @@ void bsd_os_errno_set(int err_code)
 	}
 }
 
+s64_t bsd_os_uptime_get(void)
+{
+    return k_uptime_get();
+}
+
+s64_t bsd_os_uptime_delta(s64_t *reftime)
```

Comment:
Is this for convenience or do we really need both the delta and the absolute?

Reply:
It's for convenience but I think it's goot to have both anyway.